app/controllers/idv/verify_documents_controller.rb

```diff
@@ -0,0 +1,60 @@
+# API to submit image urls for processing to verification vendor and poll for results
+
+module Idv
+  class VerifyDocumentsController < ApplicationController
+    include ApplicationHelper
+
+    before_action :render_404_if_disabled
+
+    respond_to :json
+
+    def index
+      analytics.track_event(Analytics::IDV_DOC_AUTH_DOCUMENT_STATUS)
+
+      render json: { status: :success }
```

Comment:
this is temporary, right? we still need to load the status to check here

Reply:
yep all temporary...generally WIP is not up for comments :-)